### ISSUE TITLE
No mutation rate for OutOfAfricaArchaicAdmixture_5R19

### ISSUE THREAD

**Issue by silastittes** [popsim-consortium/analysis2#117]
Production pipeline is failing simulations. I think because the demographic model we're currently using doesn't have a mutation rate associated with it. Is this an issue for the catalogue itself? Happy to add an issue there too.

The tiny config uses `OutOfAfrica_3G09`. Perhaps we should just switch to that demography?

```
import stdpopsim
species = stdpopsim.get_species("HomSap")
species.get_demographic_model("OutOfAfricaArchaicAdmixture_5R19").mutation_rate #returns None
```

**Comment by silastittes** [comment]
Reading more, seems like the lack of mutation rate in this model is intentional, but still problematic for our purposes here.

**Comment by lntran26** [comment]
I ran into this in #109 . The first paper used `OutOfAfricaArchaicAdmixture_5R19` and the mean species mutation rate. I believe we decided to stick with `OutOfAfricaArchaicAdmixture_5R19` instead of `OutOfAfrica_3G09` to be consistent with the first paper. In my run I added a little if-else loop to handle this model like so:
```
        elif wildcards.demog == 'OutOfAfricaArchaicAdmixture_5R19':
            model = species.get_demographic_model(wildcards.demog)
            mutation_rate = species.genome.mean_mutation_rate
```
Not super elegant, but it should run.

**Comment by andrewkern** [comment]
hey Silas-- so in the case where a demographic model isn't associated with a mutation rate, we should just use the species default mutation rate. so in this case that would look something like

```python
import stdpopsim as std

hs = std.get_species("HomSap")
mut_rate = hs.get_contig("1").mutation_rate # or whatever contig
```

**Comment by silastittes** [comment]
Sounds good. Thanks team!

**Comment by silastittes** [comment]
closed by #114
